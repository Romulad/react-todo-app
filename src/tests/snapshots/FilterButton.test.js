import renderer from 'react-test-renderer';
import FilterButton from '../../components/FilterButton';


test("FilterButton renders correctly", ()=>{
    const component = renderer.create(
        <FilterButton  name="All" isActive="true"></FilterButton>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
