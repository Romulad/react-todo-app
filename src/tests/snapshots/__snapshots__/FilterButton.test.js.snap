// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterButton renders correctly 1`] = `
<button
  aria-pressed="true"
  className="btn toggle-btn"
  onClick={[Function]}
  type="button"
>
  <span
    className="visually-hidden"
  >
    Show 
  </span>
  <span>
    All
  </span>
  <span
    className="visually-hidden"
  >
     tasks
  </span>
</button>
`;
